### DISCUSSION TITLE
Requires JavaFX / Requires Full JDK Radiobuttons are somehow misleading.

### DISCUSSION THREAD

**Discussion by timoreichen** [shannah/jdeploy#127] [Ideas]
Selecting these options finally does ".stripping it from the build jar"..? Shouldnt the title for this Radio-Buttons not be something like: "Javafx - managed by installer" / "Jdk - managed by installer".  

Or:
Managed by installer [x] Javafx / [x] Jdk (Selecting will exclude it from packaging).. Anyway, somehow i think its misleading and needs an improvemnt

**Comment by shannah** [comment]
When you read the current title "Requires Full JDK", what does it mean to you?
Same question for "Requires JavaFX"?

I'll consider the change in wording.  Not sure yet whether I agree this is more clear or not.

**Comment by timoreichen** [comment]
Fully understandable :-) and no dealbreaker. I read: let jdeploy include it in the deployment-process. But then afterwards i read, its stripped out ;-) I just stumbled over it.  I will think over it again/also and come back if i find a better wording.
